src/pages/progress/index.tsx

```diff
@@ -180,7 +183,12 @@ export const ProgressPage: FC<ProgressPageProps> = ({ offrampingState }) => {
     <BaseLayout
       main={
         <main>
-          <ProgressContent currentPhase={currentPhase} currentPhaseIndex={currentPhaseIndex} message={message} />
+          <ProgressContent
+            currentPhase={currentPhase}
+            currentPhaseIndex={currentPhaseIndex}
+            message={message}
+            network={selectedNetwork}
```

Comment:
we don't have to pass `selectedNetwork` to this component. We can just use `useNetwork` inside of ProgressContent to reduce props drilling

Reply:
Changed.